src/app/molecules/message/Message.jsx

```diff
@@ -228,10 +233,11 @@ MessageOptions.propTypes = {
 
 function Message({
   avatar, header, reply, content, editContent, reactions, options,
+  msgType,
```

Comment:
Similarly to the question up top: should this be kept on a single line or is this worthy of being split?